app/src/main/java/org/oppia/app/onboarding/OnboardingListener.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.onboarding
+
+interface OnboardingListener {
```

Comment:
Please add documentation to class & methods. Ditto elsewhere in this PR.

Also, please run ktlint on your PRs & fix corresponding linter errors.

Reply:
Okay!